src/shared/components/query/StudyListLogic.ts

```diff
@@ -299,6 +299,13 @@ export class FilteredCancerTreeView {
             if (this.store.isDeletedVirtualStudy(study.studyId)) {
                 return true;
             }
+            if (
+                getServerConfig().skin_home_page_show_unauthorized_studies &&
```

Comment:
i think if readPermission is false we can ALWAYS deactivate?  i.e. no need to check the configuration flag?   also how come we check for studyId (just curious)?

Reply:
Correct. I modified it to only check if the readPermission is false.
Regarding studyId, I copied the entire condition code from the StudyList. I don't remember the reason I put it there and why I was getting error otherwise, but I left it here as well just as double checking. I removed it and tested it.